docs/docs/gestures/gesture-detector.md

```diff
@@ -42,6 +42,10 @@ Starting with Reanimated 2.3.0 Gesture Handler will provide a [StateManager](/do
 
 This parameter allows to specify which `userSelect` property should be applied to underlying view. Possible values are `"none" | "auto" | "text"`. Default value is set to `"none"`.
 
+### `enableContextMenu(value: boolean)` (**web only**)
```

Comment:
```suggestion
### `enableContextMenu(value: boolean)` (Web only)
```

Reply:
In order to stay consistent, we will have to change it also in `userSelect` property. We could also go through our docs and change all occurrences (it appears that there are only 2 beside those mentioned here). What do you think?

Reply:
yeah, change it everywhere to one format please

Reply:
Done in [f772fba](https://github.com/software-mansion/react-native-gesture-handler/pull/2790/commits/f772fbac11af58f89f051c0e901626bd238eb1c9)